common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -560,10 +557,6 @@ def should_force_account_creation():
             if should_force_account_creation():
                 return dispatch_to_register()
             return dispatch_to_login()
-        elif auth_entry == AUTH_ENTRY_REGISTER or auth_entry == AUTH_ENTRY_REGISTER_V2:
```

Comment:
Please reverify this change. May be we need to update this sanity instead of removing it.